HGustavs/LenaSYS
# G1 2021 w18 #10466

* Will overrule localstorage variant if we use hash url.
* Will keep expiry time if we change from localstorage variant to hash url

**To test:** 
- Change TimeToLive at line 1181 and 1212 to appropriate time for testing. 
- Go to useranswer and test different hashes etc.
- Delete in localstorage and see if it sets correctly
